src/components/App/DataManager.ts

```diff
@@ -144,15 +144,19 @@ export class DataManager extends EventEmitter<Events> {
     });
   }
 
-  private setupServicesEventHandlers(stream: IEventStream, frame: StoreFrame) {
+  private setupServicesEventHandlers(
+    stream: IEventStream,
+    frame: StoreFrame,
+    filters?: Filters,
+  ) {
     stream.on(EventStreamEventKind.Service, (svcChange: ServiceChange) => {
       frame.applyServiceChange(svcChange.service, svcChange.change);
     });
 
     stream.on(EventStreamEventKind.Flows, (flows: HubbleFlow[]) => {
-      const { flowsDiffCount } = frame.addFlows(flows);
+      const { flowsDiffCount } = frame.addFlows(flows, filters);
```

Comment:
I know that we have already talked about this filtering, but now I understood its wrong. Passing filters to  `Frame::addFlows` also means passing it to `InteractionStore::addFlows` which is not good as filtering logic is blurring across multiple stores and modules. Currently I think its better to keep it in one place. We have `Frame::filter` and call it here, in `DataManager`, so I suggest to return to one-argument version of `addFlows`, pass already filtered flows into it. Flows filtering is better to do here, in `DataManager`. It could be like that:
```ts
...
const filteredFlows = flows.filter(f => filtering.filterFlow(f, filters));
const { ... } = frame.addFlows(filteredFlows);
...
```